FWCore/Framework/src/Worker.h

```diff
@@ -875,10 +906,21 @@ namespace edm {
     };
   }  // namespace workerhelper
 
+  template <typename T>
+  void Worker::prefetchAsync(WaitingTask* iTask,
+                             ServiceToken const& token,
+                             ParentContext const& parentContext,
+                             typename T::TransitionInfoType const& transitionInfo,
+                             Transition iTransition) {
+    Principal const& principal = transitionInfo.principal();
+    prepareToPrefetch(iTask, parentContext, principal);
```

Comment:
Done. I moved the incrementing and decrementing into Worker.h in this function. It is much easier to read that way and I like it better. Originally, I was just trying to put as much as possible into the .cc file because I know Worker.h gets included in every module everywhere. I was just trying to minimize its content, but it makes it so hard to read it's not worth doing this case.